index.html

```diff
@@ -49,6 +49,7 @@
     <div id='nav'>
       <div class='container center'>
         <a href="#contribute">Contribute</a>
+        <a href="#ci">CI</a>
```

Comment:
I think the top navbar is crowded as it is, and we should rather remove items than add new ones (the centering makes wraparound painful on "Plan checker" etc). In this case I think Contribute is fine as a toplevel nav item, CI is a subsection to Contribute IMO.